service/templates/distros.html

```diff
@@ -961,7 +961,7 @@ <h2 style="display:inline-block; padding-right:15px">Configure
             </p>
             <p class="distro-checkbox checkbox">
               <input type="checkbox" ng-model="activeDistro.disabled">
-              Disable scheduling this distro. Tasks already in the schedule will be dispatched.
+              Disable scheduling this distro. Tasks already in the schedule will be removed and re-scheduled upon re-enable.
```

Comment:
s/schedule/queue

Also I think you can drop "and re-scheduled upon re-enable", since that isn't necessarily true. For example, the tasks might be deactivated in the intervening time.